cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/NodeInlineContentHandler.java

```diff
@@ -83,12 +83,13 @@
 import ca.nrc.cadc.uws.Parameter;
 import ca.nrc.cadc.uws.web.InlineContentException;
 import ca.nrc.cadc.uws.web.InlineContentHandler;
+import ca.nrc.cadc.uws.web.UWSInlineContentHandler;
 import ca.nrc.cadc.vos.Node;
 import ca.nrc.cadc.vos.NodeParsingException;
 import ca.nrc.cadc.vos.NodeReader;
 import ca.nrc.cadc.vos.NodeWriter;
 
-public class NodeInlineContentHandler implements InlineContentHandler
+public class NodeInlineContentHandler implements UWSInlineContentHandler
```

Comment:
the param list and job info vars should be removed, plus methods related to them

Reply:
rename to RecursiveSetNode{InlineContent}Handler?